corehq/apps/domain/deletion.py

```diff
@@ -188,15 +190,17 @@ def delete_all_cases(domain_name):
     logger.info('Deleting cases...')
     case_ids = iter_ids(CommCareCase, 'case_id', domain_name)
     for chunk in chunked(case_ids, 1000, list):
-        CommCareCase.objects.hard_delete_cases(domain_name, chunk)
+        CommCareCase.objects.hard_delete_cases(domain_name, chunk, publish_changes=False)
+        case_adapter.bulk_delete(chunk)
```

Comment:
This should also delete from `case_search` index if case_search is enabled on domain.